From 0bf4fb5a1f25649d467869c4fac8d4c927a423f6 Mon Sep 17 00:00:00 2001
From: Sugar Zhang <sugar.zhang@rock-chips.com>
Date: Thu, 9 Jul 2015 16:17:50 +0800
Subject: [PATCH] ASoC: hdmi_spdif: support format SNDRV_PCM_FMTBIT_S24_LE

because of DMA_SLAVE_BUSWIDTH_4_BYTES, we support SNDRV_PCM_FMTBIT_S24_LE
which store data in low three bytes. not support SNDRV_PCM_FMTBIT_S24_3LE

Signed-off-by: Sugar Zhang <sugar.zhang@rock-chips.com>
---
 sound/soc/codecs/hdmi_spdif.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/codecs/hdmi_spdif.c b/sound/soc/codecs/hdmi_spdif.c
index a820a8e13f98..792263cffa19 100755
--- a/sound/soc/codecs/hdmi_spdif.c
+++ b/sound/soc/codecs/hdmi_spdif.c
@@ -26,7 +26,7 @@
 #define DRV_NAME "spdif-dit"
 
 #define STUB_RATES	SNDRV_PCM_RATE_8000_192000
-#define STUB_FORMATS	SNDRV_PCM_FMTBIT_S16_LE
+#define STUB_FORMATS	(SNDRV_PCM_FMTBIT_S16_LE | SNDRV_PCM_FMTBIT_S24_LE)
 
 
 static struct snd_soc_codec_driver soc_codec_spdif_dit;
-- 
2.35.3

